profiles/core/inputs.tf

```diff
@@ -1,237 +1,243 @@
-variable deploy_linkerd { 
+variable "deploy_linkerd" {
```

Comment:
not sure why your local formatting is adding these extra quotes around variable names, but can we remove them?